gradle.properties

```diff
@@ -1,5 +1,8 @@
 kapt.use.worker.api=true
-
-bootVersion=1.5.14.RELEASE
+bootVersion=1.5.19.RELEASE
```

Comment:
Have to manage this here because `spinnaker.version("springBoot")` can't be used on a Gradle plugin.